pdal/pipeline.py

```diff
@@ -28,7 +28,10 @@ def __init__(
         spec: Union[None, str, Sequence[Stage]] = None,
         arrays: Sequence[np.ndarray] = (),
         loglevel: int = logging.ERROR,
+        json: str = None
     ):
+        if json:
+            spec = json
```

Comment:
Perhaps raise a ValueError if both `spec` and `json` are passed?

Reply:
I did so in 81203f1f1ae7ffed41bcdcaf057c4897f1a83b38